app/models/vehicles.py

```diff
@@ -73,7 +72,7 @@ def get(vehicle_id):
     if not result:
         raise ErrVehicleNotFound(f"no vehicle found for the provided id: {vehicle_id}")
 
-    return result[0]
+    return result.pop()
```

Comment:
I noticed in consistency here, sometimes the mode has `vehicle_id` and sometimes `id` also why do you need to change those models? I think if this PR is only about services we should limit the scope to only services. 
To make the PR smaller can you just checkout those models from master so they don't get included in the PR